### PR TITLE
TG-442 Add Coverity scan

### PR DESCRIPTION
**Pull request by zemanlx**
Coverity scan will run as separate job in second stage. Ii will terminate
itself as soon as possible if build is not triggered by cron. Therefore
it requires to have cron job setup (Branch:"develop" Intrerval:"Daily" Options:"Do not run if there has been a build in the last 24h") in settings of diffblue/cbmc and
environment variable `COVERITY_SCAN_TOKEN` in the same project settings.

image

Example of **cron** job - https://travis-ci.org/zemanlx/cbmc/jobs/269950577
Example of **non-cron** job - https://travis-ci.org/zemanlx/cbmc/jobs/269949324

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -151,6 +151,47 @@ jobs:
...
+        - |
+          if [[ "${TRAVIS_EVENT_TYPE}" != "cron" ]]
+          then
+            echo "This is not a cron build and build is not needed."
+            travis_terminate 0
+          fi
+        - mkdir bin ; ln -s /usr/bin/gcc-5 bin/gcc
+        - echo -n | openssl s_client -connect scan.coverity.com:443 |
+          sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' |
+          sudo tee -a /etc/ssl/certs/ca-certificates.crt
```
Comment (tautschnig) [inline .travis.yml]: Why is this necessary? trusty does ship the required CA certificate.

Comment (zemanlx) [reply .travis.yml]: It was recommended by coverity, but I will try it without it.

**Comment by zemanlx** [comment]
@tautschnig it is fixed now, can you please re-review?
